From: Gunnar Hjalmarsson <gunnarhj@ubuntu.com>
Date: Tue, 26 Mar 2013 17:04:18 +0100
Subject: Adapt for Ubuntu language handling

This patch was taken from gnome-control-center 52_region_language.patch,
where relevent language code used to live in 3.6

Description: Adapts the region capplet and the language chooser in the user accounts capplet
Forwarded: https://bugzilla.gnome.org/695939, https://bugzilla.gnome.org/695940

 * Ubuntu isn't shipped with all languages installed and all locales generated.
   Instead the users install the needed language pack(s), and the corresponding
   locales are generated.

 * The 'common_name' field taken into account when parsing the iso-codes XML files
   for language names.
---
 libgnome-desktop/gnome-languages.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgnome-desktop/gnome-languages.c b/libgnome-desktop/gnome-languages.c
index e27ccbd..d69b0c5 100644
--- a/libgnome-desktop/gnome-languages.c
+++ b/libgnome-desktop/gnome-languages.c
@@ -1140,7 +1140,7 @@ gnome_get_language_from_locale (const char *locale,
                                         translated_territory);
         }
 
-        language_name_get_codeset_details (locale, &langinfo_codeset, &is_utf8);
+        //language_name_get_codeset_details (locale, &langinfo_codeset, &is_utf8);
 
         if (codeset_code == NULL && langinfo_codeset != NULL) {
                 codeset_code = g_strdup (langinfo_codeset);
